edgelet/edgelet-test-utils/Cargo.toml

```
@@ -12,7 +12,7 @@ http = "0.2"
 hyper = "0.14"
 nix = "0.24"
 serde = "1"
-tokio = "1"
+tokio = { version = "1", features = ["parking_lot"] }
```

```suggestion
tokio = "1"
```
This crate does not need any additional `tokio` features.